lib/location/method_cellular.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <zephyr.h>
+#include <logging/log.h>
+
+#include <modem/location.h>
+#include <modem/lte_lc.h>
+#include <net/multicell_location.h>
+
+#include "loc_core.h"
+
+LOG_MODULE_DECLARE(location, CONFIG_LOCATION_LOG_LEVEL);
+
+extern location_event_handler_t event_handler;
+extern struct loc_event_data current_event_data;
+
+struct k_work_args {
+	struct k_work work_item;
+	struct loc_cellular_config cellular_config;
+};
+
+static struct k_work_args method_cellular_positioning_work;
+
+static K_SEM_DEFINE(cellmeas_data_ready, 0, 1);
+
+static struct lte_lc_ncell neighbor_cells[CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS];
+static struct lte_lc_cells_info cell_data = {
+	.neighbor_cells = neighbor_cells,
+};
+static bool running;
+
+void method_cellular_lte_ind_handler(const struct lte_lc_evt *const evt)
+{
+	switch (evt->type) {
+	case LTE_LC_EVT_NEIGHBOR_CELL_MEAS: {
```

Comment:
What about supporting single cell positioning?

nRF Cloud offers single and multi cell as options -- there is apparently a business case for this.  So, supporting it as an option is important.

Does not the concept of accuracy in this library specifically enable this?

Reply:
This discussion has been handled elsewhere.